src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProvider.java

```diff
@@ -73,8 +73,6 @@
 @Component
 public class VMWareProvider implements CloudProvider {
 
-    private final Logger logger = Logger.getLogger(VMWareProvider.class);
-
     private final static String IMAGE_DELIMITER = "/";
 
     private final static String RANDOM_HOST = "*";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)